devito/compiler.py

```diff
@@ -12,7 +12,7 @@
 from codepy.toolchain import GCCToolchain
 
 from devito.exceptions import CompilationError
-from devito.logger import log, warning
+from devito.logger import debug, warning
```

Comment:
Why the change from 'log' to 'debug'?

Reply:
to avoid pollution to ``stdout``. I think the average user doesn't need to see a jit-compilation related string.